schema/crdb/dbinit.sql

```diff
@@ -3943,6 +3944,22 @@ CREATE TABLE IF NOT EXISTS omicron.public.inv_sled_agent (
     -- similar to `usable_hardware_threads` and friends above.
     cpu_family omicron.public.sled_cpu_family NOT NULL,
 
+    -- The path to the boot disk image file.
+    measurement_manifest_boot_disk_path TEXT NOT NULL,
+    -- The source of the zone manifest on the boot disk: from installinator or
+    -- sled-agent (synthetic). NULL means there is an error reading the zone manifest.
+    measurement_manifest_source omicron.public.inv_zone_manifest_source,
```

Comment:
I'd probably make this an enum and explicitly include `ErrorReadingZoneManifest` or something similar.